toolchain/parser/parse_tree.h

```diff
@@ -166,7 +159,7 @@ class ParseTree {
                       int subtree_size_arg)
         : kind(k), token(t), subtree_size(subtree_size_arg) {}
 
-    // TODO: Parser2 only uses this construct. Can remove the other if we
+    // TODO: Parser only uses this construct. Can remove the other if we
```

Comment:
Since this removes the other parser, remove the other here?

Reply:
Thanks, removed